app.py

```diff
@@ -22,7 +30,17 @@ def beep(repeat):
     body = sel.find_element_by_tag_name('body')
     if body.get_attribute('class') == 'queue challenge':
         queue = True
-        beep(5)
+        say("ps5 queue is active")
+    if body.get_attribute('class') == 'softblock':
+        say("captcha-challenge block")
+        time.sleep(CAPTCHA_TIME_ALLOWANCE) # allow 20 seconds to solve captcha before attempting to refresh
     else:
-        time.sleep(randint(4, 10))
-        sel.refresh()
+        try:
+            add_to_cart_button = sel.find_element_by_class_name('add-to-cart') # first add-to-cart button
```

Comment:
Unsure this would work in practice - the script quits when the queue is reached, and would require solving a captcha as part of entering the queue if you were hoping to have it continue to refresh the queue page to wait for shop availability. Also, no guarantee that the button would be available from the landing page of the shop, since the script does not go directly to a listing page for a particular console edition. Consider separating the TTS functionality from the add_to_cart functionality so it can be tested separately with tomorrow's drop.